requirements-dev.in

```diff
@@ -5,3 +5,4 @@ pre-commit==3.5.0
 pyright==1.1.337
 pytest==7.4.2
 python-box==7.1.1
+werkzeug==3.0.3
```

Comment:
does this also need to be in the dev dependencies?

Reply:
oh probably not, good point